packages/itwin/grouping-mapping-widget/src/widget/components/Mapping.tsx

```diff
@@ -14,24 +14,22 @@ import {
 } from "@itwin/itwinui-icons-react";
 import {
   Button,
-  ButtonGroup,
   DropdownMenu,
   IconButton,
   MenuItem,
-  Table,
+  Surface,
 } from "@itwin/itwinui-react";
-import type { CellProps } from "react-table";
-import React, { useCallback, useEffect, useMemo, useState } from "react";
+import React, { useCallback, useEffect, useState } from "react";
 import type { CreateTypeFromInterface } from "../utils";
-import { handleError, onSelectionChanged, WidgetHeader } from "./utils";
+import { EmptyMessage, handleError, LoadingOverlay, onSelectionChanged, WidgetHeader } from "./utils";
 import "./Mapping.scss";
 import DeleteModal from "./DeleteModal";
 import { Groupings } from "./Grouping";
 import MappingAction from "./MappingAction";
 import { MappingImportWizardModal } from "./MappingImportWizardModal";
 import { useMappingClient } from "./context/MappingClientContext";
 import type { Mapping } from "@itwin/insights-client";
-import { BlockingOverlay } from "./BlockingOverlay";
+import { HorizontalTile } from "./HorizontalTile";
```

Comment:
Can you bring back the blocking overlay?

Reply:
Will do.